qiskit/chemistry/results/electronic_structure_result.py

```diff
@@ -131,73 +131,84 @@ def reverse_dipole_sign(self, value: bool) -> None:
+            edm = [cast(DipoleTuple, tuple(-1 * x if x is not None else None for x in dip))
+                   for dip in edm]
+        return [_dipole_tuple_add(dip, self.nuclear_dipole_moment) for dip in edm]
 
     @property
-    def dipole_moment_in_debye(self) -> Optional[DipoleTuple]:
+    def dipole_moment_in_debye(self) -> Optional[List[DipoleTuple]]:
         """ Returns dipole moment in Debye """
         dipm = self.dipole_moment
         if dipm is None:
             return None
-        dipmd0 = dipm[0]/QMolecule.DEBYE if dipm[0] is not None else None
-        dipmd1 = dipm[1]/QMolecule.DEBYE if dipm[1] is not None else None
-        dipmd2 = dipm[2]/QMolecule.DEBYE if dipm[2] is not None else None
-        return dipmd0, dipmd1, dipmd2
+        dipmd = []
+        for dip in dipm:
+            dipmd0 = dip[0]/QMolecule.DEBYE if dip[0] is not None else None
+            dipmd1 = dip[1]/QMolecule.DEBYE if dip[1] is not None else None
+            dipmd2 = dip[2]/QMolecule.DEBYE if dip[2] is not None else None
+            dipmd += [(dipmd0, dipmd1, dipmd2)]
+        return dipmd
 
     @property
-    def electronic_dipole_moment(self) -> Optional[DipoleTuple]:
+    def electronic_dipole_moment(self) -> Optional[List[DipoleTuple]]:
         """ Returns electronic dipole moment """
-        return _dipole_tuple_add(self.computed_dipole_moment,
-                                 _dipole_tuple_add(self.ph_extracted_dipole_moment,
-                                                   self.frozen_extracted_dipole_moment))
+        return [_dipole_tuple_add(comp_dip, _dipole_tuple_add(ph_dip, frozen_dip)) for
+                comp_dip, ph_dip, frozen_dip in zip(self.computed_dipole_moment,
+                                                    self.ph_extracted_dipole_moment,
+                                                    self.frozen_extracted_dipole_moment)]
 
     @property
-    def computed_dipole_moment(self) -> Optional[DipoleTuple]:
+    def computed_dipole_moment(self) -> Optional[List[DipoleTuple]]:
         """ Returns computed electronic part of dipole moment """
         return self.get('computed_dipole_moment')
 
     @computed_dipole_moment.setter
-    def computed_dipole_moment(self, value: DipoleTuple) -> None:
+    def computed_dipole_moment(self, value: List[DipoleTuple]) -> None:
         """ Sets computed electronic part of dipole moment """
         self.data['computed_dipole_moment'] = value
 
     @property
-    def ph_extracted_dipole_moment(self) -> Optional[DipoleTuple]:
+    def ph_extracted_dipole_moment(self) -> Optional[List[DipoleTuple]]:
         """ Returns particle hole extracted part of dipole moment """
         return self.get('ph_extracted_dipole_moment')
 
     @ph_extracted_dipole_moment.setter
-    def ph_extracted_dipole_moment(self, value: DipoleTuple) -> None:
+    def ph_extracted_dipole_moment(self, value: List[DipoleTuple]) -> None:
```

Comment:
I get this can be a list, but only ever one value is extracted from the fermionic operator (same applies to other quantities here too)

Reply:
this is to remain consistent with the electronic structure approach, can be extended